Task-43333: Regression - bad display(alignment-UI glitches) of task changes in change history

montassarhamadi opened the pull request:
Problem: a bad alignment of the task changes in the task's changes tab is displayed.
Fix: Task changes should be well displayed with aligned.

montassarhamadi committed:
Used the features of Vuetify spacing instead of classes declared in tasks.less file

rdenarie reviewed (commented):
deleted comment

rdenarie reviewed (changes requested):
Hello
This PR seems not ok.
In the associated task, there is 3 differents PR to cherry pick.
As I can see you get only the last one

montassarhamadi committed:
Task-43333: Regression - bad display(alignment-UI glitches) of task changes in change history.

Problem: a bad alignment of the task changes in task's changes tab is displayed.
Fix: Task changes should be well displayed with aligned.

Montassar-Hamadi commented:
@rdenarie the first and the 2nd PR changes are combined into this PR's first commit, the 2nd commit is related to the 3rd PR changes.
